### PR TITLE
Support bem-xjst 8.x

### LINKED ISSUE #1942: Support bem-xjst 8.x

### PR DESCRIPTION
**Pull request by tadatuta**
Closes #1942 

Cherry-pick of https://github.com/bem/bem-components/pull/1996 by @miripiruni

---

Showcase is [available](http://bem.github.io/reports/6619/showcase/showcase.html)

### CODE REVIEW

**Comment by miripiruni** [comment]
👍 

Ready to merge?

File: .jshint-groups.js [hunk 1]
```diff
@@ -62,16 +62,19 @@ module.exports = {
                     'applyCtx',
                     'applyNext',
                     'attrs',
+                    'addAttrs',
                     'bem',
                     'block',
                     'cls',
                     'content',
```
Comment (qfox) [inline .jshint-groups.js:69]: appendContent?

File: .jshint-groups.js [hunk 2]
```diff
@@ -62,16 +62,19 @@ module.exports = {
...
                     'elem',
                     'js',
+                    'addJs',
                     'local',
                     'match',
                     'mix',
+                    'addMix',
                     'mod',
                     'mode',
                     'tag',
```
Comment (qfox) [inline .jshint-groups.js:82]: Guess we should drop `once` and `extend` probably.

File: common.blocks/menu/menu.bemhtml.js [hunk 3]
```diff
@@ -46,10 +46,12 @@ block('menu')(
             attrs['aria-disabled'] = 'true' :
             attrs.tabindex = 0;
 
-        return attrs;
+        // extend in backwards order:
+        // bemjson have more priority
```
Comment (qfox) [inline common.blocks/menu/menu.bemhtml.js]: bemjson haS

File: .jshint-groups.js (hunk 2 above)
Comment (miripiruni) [reply .jshint-groups.js:82]: `extend` continue exists

File: .jshint-groups.js (hunk 1 above)
Comment (miripiruni) [reply .jshint-groups.js:69]: …and prependContent

**Pull request merged**